src/utils/useAdBreak.ts

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect, useContext } from 'react';
+import { PlayerContext } from 'jwplayer/players/shared/PlayerContext';
+import JWEvents from 'jwplayer/players/shared/JWEvents';
+
+export default function useAdBreak(): boolean {
+	const [adBreak, setAdBreak] = useState(false);
+	const { player } = useContext(PlayerContext);
+
+	useEffect(() => {
+		player?.on(JWEvents.AD_BREAK_START, () => {
+			console.log('AD_BREAK_START');
```

Comment:
This ad_break_start event does not seem to fire whenever the ad plays

Reply:
We may not be triggering this with the way we're loading ads.

Reply:
Got it. Maybe that'll get fixed if we utilize the built in JW API ad methods sometime in the future.